operators/hpe-csi-operator/deploy/role.yaml

```diff
@@ -102,6 +102,10 @@ rules:
   - apiGroups: ["storage.hpe.com"]
     resources: ["hpenodeinfos"]
     verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
+# Need permissions to let csi-driver load/unload/update hpevolumeinfos resources
+  - apiGroups: ["storage.hpe.com"]
```

Comment:
we need some `ClusterRole` , `ClusterRoleBinding` RBAC's for this CRD ... Any idea of where are those in this installer ?

Reply:
role.yaml and role_binding.yaml has the RBAC for the CRD.
I have added for hpevolumeinfo L106 to L109 in role.yaml , you can verify .Let me know if we need to add at any other place